package.json

```diff
@@ -388,7 +388,8 @@
     "webtorrent": "^0.114.0"
   },
   "overrides": {
-    "loader-utils": "2.0.4",
-    "json5": "2.2.3"
+    "loader-utils": "2.0.3",
```

Comment:
Did this mean to downgrade `loader-utils` from 2.0.4 to 2.0.3?

Reply:
good catch - don't think this is intentional

Reply:
@petemill https://github.com/brave/brave-core/pull/19347 fixes this